usaspending_api/search/v2/views/spending_by_award.py

```
@@ -283,34 +294,80 @@ def populate_response(self, results: list, has_next: bool) -> dict:
+            results = [
+                self.date_to_epoch_millis(queryset[0].get(self.get_sort_by_fields()[0])),
+                float(queryset[0].get("total_obligation")),
+                queryset[0].get("award_id"),
+            ]
+            search = (
                 AwardSearch()
                 .filter(filter_query)
                 .sort(*sorts)
-                .extra(search_after=[self.last_value, self.last_id])[0 : self.pagination["limit"]]
+                .extra(search_after=[*results])[0 : self.pagination["limit"]]
             )
-            if self.last_value and self.last_id
-            else (
+            response = search.handle_execute()
+            return response
+        search = (
+            (
                 AwardSearch()
                 .filter(filter_query)
-                .sort(*sorts)[
-                    ((self.pagination["page"] - 1) * self.pagination["limit"]) : (
-                        ((self.pagination["page"] - 1) * self.pagination["limit"]) + self.pagination["limit"]
-                    )
-                ]
+                .sort(*sorts)
+                .extra(
+                    search_after=[
+                        self.date_to_epoch_millis(self.last_record_sort_value),
+                        self.last_record_tie_break_value,
+                        self.last_record_unique_id,
+                    ]
+                )[0 : self.pagination["limit"]]
             )
+            if self.last_record_sort_value is not None and self.last_record_unique_id is not None
+            else (AwardSearch().filter(filter_query).sort(*sorts)[record_num : record_num + self.pagination["limit"]])
         )
         response = search.handle_execute()
 
         return response
 
-    def construct_es_reponse(self, response) -> dict:
+    def construct_es_response(self, response) -> dict:
         results = []
+        if response is None:
+            return {"There was an error connecting to the Elasticsearch cluster"}
```

In general we should not return lower level details about backend problems through the API. 
- If this is a hint to the developer, it should be logged, so the developer can inspect logs
- For the API caller, it should probably throw an exception with a friendly non-detailed error message that something went wrong (or however we would return a `500` code with generic error message)

---

Removed